client/Android/Studio/freeRDPCore/src/main/cpp/android_cliprdr.c

```diff
@@ -422,11 +425,22 @@ android_cliprdr_server_format_data_response(CliprdrClientContext* cliprdr,
 		JNIEnv* env;
 		jstring jdata;
 		jboolean attached;
-		formatId = ClipboardRegisterFormat(afc->clipboard, "UTF8_STRING");
 		data = (void*)ClipboardGetData(afc->clipboard, formatId, &size);
+
```

Comment:
should have been sufficient to replace `formatId` with `CF_TEXT` `UTF8_STRING` (that one is a leftover from 2.0)
`ClipboardInitSynthesizers` (winpr) does some magic on text formats to always have these available and on linux/mac os we have only utf-8 (at least nowadays) so local encoding issues do not apply